drake/systems/plants/collision/ResultCollector.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+
+#include "drake/systems/plants/collision/DrakeCollision.h"
+#include "ResultCollector.h"
```

Comment:
- The related-header must appear as the first (line 1) include.
- All includes must use the full path ("drake/...").

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes